huaweicloud/services/css/resource_huaweicloud_css_cluster.go

```diff
@@ -429,59 +387,77 @@ func ResourceCssCluster() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
-
 						"name": {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
-
 						"type": {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
-
 						"availability_zone": {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
-
 						"status": {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
-
 						"spec_code": {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+						"ip": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"resource_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
 					},
 				},
 			},
-
-			"created": {
```

Comment:
this attribute should be kept, if it has been used, an error will be a occured it is deleted, it only removed from the doc